lib/browser-runtime.ts

```diff
@@ -286,7 +287,17 @@ function createStepDescription({
 }
 
 function createFeature(context: CompositionContext, feature: messages.Feature) {
-  describe(feature.name || "<unamed feature>", () => {
+  const suiteOptions = collectTagNames(feature.tags)
+    .filter(looksLikeOptions)
+    .map(tagToCypressOptions)
+    .filter((tag) => {
+      return Object.keys(tag).every((key) =>
+        SUITE_CONFIGURATION_OPTIONS.includes(key)
+      );
+    })
+    .reduce(Object.assign, {});
+
+  describe(feature.name || "<unamed feature>", suiteOptions, () => {
```

Comment:
There's another instance where `describe` is invoked and that's with rules (see `createRule` in the same file). I that if this were to be supported, then one ought to be able to tag rules in the same way.

Reply:
Done cfe822c70a5a9d0a5c68a231a129283dce40cdae.

Reply:
Does configuration overriding work? Eg

```gherkin
@testIsolation(false)
Feature: a feature
  @testIsolation(true)
```

The gherkin parser won't resolve this as tags are just strings but I guess the order of tags is being relied on here?

Reply:
I imagine that should work, yeah, and that the latest value takes precedence.